nabla-pendulum/deploy/04_deploy_curves.ts

```diff
@@ -0,0 +1,44 @@
+import { WasmDeployEnvironment } from "../../src/index";
+
+const CURVE_PARAMETERS_LOW_SLIPPAGE = {
+  alpha: 0.02,
+  beta: 0.0011,
```

Comment:
Is this considered a considerable difference between having `beta` as `0.0011` vs `0.001` as already described [here](https://www.notion.so/satoshipay/24-05-16-Nabla-Pendulum-Deployment-c7cefd58f44c428a92ceaea30c603286?pvs=4#ccc59b7a948a4e16893e7f3d854af8cc)?